test/ably/restpush_test.py

```diff
@@ -37,3 +40,40 @@ def test_admin_publish(self):
 
         response = publish(recipient, data)
         assert response.status_code == 204
+
+    # RSH1b3
+    def test_admin_device_registrations_save(self):
+        save = self.ably.push.admin.device_registrations.save
+
+        device_id = random_string(26, string.ascii_uppercase + string.digits)
+        data = {
+            'id': device_id,
+            'platform': 'ios',
+            'formFactor': 'phone',
+            'push': {
+                'recipient': {
+                    'transportType': 'apns',
+                    'deviceToken': '740f4707bebcf74f9b7c25d48e3358945f6aa01da5ddb387462c7eaf61bb78ad'
```

Comment:
Where is this value coming from? If it's reused I suggest you store it somewhere global or on a file. That way, if it changes in the future, we have to update it only in one place.

Reply:
The value comes from the documentation
https://github.com/ably/docs/blob/master/content/rest-api/index.textile